index.pug

```diff
@@ -12,32 +23,24 @@ section#institution.container-fluid.sec4
         |
         | coaching class with or without a website
     .Sec4TopRight
-       img.lazy(data-src="/assets/education/institute.svg",
-        alt="institute"
-      )
+       img.lazy(data-src="/assets/education/institute.svg", alt="institute")
```

Comment:
<img width="728" alt="Screenshot 2021-11-25 at 2 42 59 PM" src="https://user-images.githubusercontent.com/10862593/143413008-0bb2c021-aa9f-486d-9428-4f24fcba06ad.png">


Can add all the images in this folder structure.